src/fun/builtins.bend

```diff
@@ -366,21 +384,37 @@ def IO/sleep(seconds: f24) -> IO(None):
 ## File IO
 
 ### File IO primitives
+
+#{ Opens a file with with `path` being given as a string and `mode` being a string with the mode to open the file in. The mode should be one of the following: #}
```

Comment:
What happened here? It should say that the mode should be 'r', 'w', 'a', 'rw'. (actually don't remember the modes by heart, might want to check)